src/LockManager.php

```diff
@@ -104,6 +104,10 @@ public function lock($resource, $lock_wait, $ttl)
 				$this->debug('> sent lock request to '. $this->servers[$i][0] .':'. $this->servers[$i][1]);
 			}
 			
+			if (count($this->sockets) < $this->quorum) {
```

Comment:
Doing it in initSockets() would be nice, but doesn't work because the use of socket_set_nonblock() means errors don't show up until write.